gensim/test/test_sklearn_api.py

```diff
@@ -1213,5 +1214,111 @@ def testModelNotFitted(self):
         self.assertRaises(NotFittedError, phrases_transformer.transform, phrases_sentences[0])
 
 
+class TestFastTextWrapper(unittest.TestCase):
+    def setUp(self):
+        numpy.random.seed(0)
```

Comment:
`numpy.random.seed(0)` affect all interpreter in general (not only current test), that's bad practice (I think we have similar mistakes in existing tests), can you please remove all calls of `numpy.random.seed` in your code @mcemilg ?

Reply:
Okay, I removed the `numpy.random.seed` calls.